lib/LoginStrategy.ts

```diff
@@ -133,7 +135,8 @@ export class LoginStrategy extends BaseStrategy {
     }
 
     try {
-      const roleClaims = await this.getUserClaims(did)
+      const roleClaims =
+        this.strategyAddress === address ? [] : await this.getUserClaims(did)
```

Comment:
IMO, a comment here to explain why this is needed would be helpful to future readers. E.g. "getUserClaims attempts to retrieve claims from cache-server and so when the cache-server itself is using the LoginStrategy, this creates a login attempt loop. Therefore, not getting userClaims if address attempting to login is the address of the strategy." (<- Feel free to correct that if it's wrong)

Reply:
Great idea, I'm forgetting to comment that kind of stuff :) no it says everything so I'll use it :)